src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -516,16 +541,20 @@ class CosmasGoogleCloudService(private val freeBucketName: String,
                     cemetery.clearCemetery().addAllCemetery(tombs).build().toByteArray())
         } catch (e: StorageException) {
             handleStorageException(e, responseObserver)
-            return
+            return@logging
         }
         val response = CosmasProto.RestoreDeletedFileResponse.getDefaultInstance()
         responseObserver.onNext(response)
         responseObserver.onCompleted()
     }
 
     override fun changeFileId(request: CosmasProto.ChangeFileIdRequest,
-                              responseObserver: StreamObserver<CosmasProto.ChangeFileIdResponse>) {
-        LOG.info("Get request for change files ids in project={}", request.info.projectId)
+                              responseObserver: StreamObserver<CosmasProto.ChangeFileIdResponse>) = logging(
+            "changeFileId", request.info.projectId) {
+        LOG.info("""Request:
```

Comment:
It looks pretty weird, why are you logging it?

![image](https://user-images.githubusercontent.com/22072254/52640464-9134f200-2ee7-11e9-93e7-03efa326a624.png)